ci_environment/android-sdk/attributes/default.rb

```diff
@@ -23,6 +23,7 @@
 default['android-sdk']['components']     = %w(platform-tools
                                               build-tools-22.0.1
                                               android-22
+                                              sys-img-armeabi-v7a-android-22
                                               android-21
```

Comment:
@can we just remove then  sys-img-armeabi-v7a-android-22 from line 28 in order to free up space for ver. 22?